contracts/src/v0.8/dev/ocr2dr/OCR2DROracleFactory.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "./OCR2DROracle.sol";
+
+/**
+ * @title OCR2DROracle Factory
+ * @notice Creates OCR2DROracle contracts for node operators
+ */
+contract OCR2DROracleFactory {
```

Comment:
Do we need a contract factory? I can't imagine anyone creating tons of oracle contracts (at least not in the near future) and the logic here is fairly trivial. We can always add it later when there's a need.

Reply:
This is a part of CLL contracts engineering guidelines. I added it to be compliant with the guidelines.